src/com/mindscriptact/mvcExpressLogger/minimalComps/components/Mvce_Window.as

```diff
@@ -33,19 +33,14 @@ import flash.display.Shape;
 import flash.display.Sprite;
 import flash.events.Event;
 import flash.events.MouseEvent;
-import flash.text.TextFieldAutoSize;
 
 [Event(name="select", type="flash.events.Event")]
 [Event(name="close", type="flash.events.Event")]
 [Event(name="resize", type="flash.events.Event")]
 public class Mvce_Window extends Mvce_Component {
-
-	protected var _titleLeft:String;
-	protected var _titleRight:String;
-
+	protected var _title:String;
 	protected var _titleBar:Mvce_Panel;
-	protected var _titleLeftLabel:Mvce_Label;
```

Comment:
any reasons why _titleRightLabel:Mvce_Label; _titleLeftLabel:Mvce_Label;  are removed?
 I do agree that it is lazy way to do stuff... :) I hacked them in instead of implementing it properly... but they are still needed.